### PR TITLE
"Lizard" sprite files renamed to "Unathi"

### PR DESCRIPTION
**Pull request by WickedCybs**
Trying to quickly search up Unathi when I'm doing something for them only to remember the sprites for their appearance were under r_lizard was always annoying. Especially considering most stuff that's Unathi related is labelled as "Unathi."

I also renamed the "Sogtail" files within the Unathi tail dmi to "Unathtail"

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

File: html/changelogs/wickedcybs_rename.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: WickedCybs
+
+delete-after: True
+
+changes:
+  - tweak: "Renamed the 'Lizard' sprite dmi's to 'Unathi'. This is an under the hood change that should not affect anything."
```
Comment (SleepyGemmy) [inline html/changelogs/wickedcybs_rename.yml:6]:
```suggestion
  - backend: "Renamed the 'Lizard' sprite dmi's to 'Unathi'. This is an under the hood change that should not affect anything."
```

File: code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 	category_name = "Unathi"
 	bodytype = BODYTYPE_UNATHI
-	icobase = 'icons/mob/human_races/unathi/r_lizard.dmi'
-	deform = 'icons/mob/human_races/unathi/r_def_lizard.dmi'
+	icobase = 'icons/mob/human_races/unathi/r_unathi.dmi'
+	deform = 'icons/mob/human_races/unathi/r_def_unathi.dmi'
 	preview_icon = 'icons/mob/human_races/unathi/unathi_preview.dmi'
 	bandages_icon = 'icons/mob/bandage.dmi'
-	tail = "sogtail"
+	tail = "unathtail"
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm:11]:
```suggestion
	tail = "unathitail"
```

File: html/changelogs/wickedcybs_rename.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  - tweak: "The Unathil tail sprite files don't reference their old name of 'Soghun' anymore."
```
Comment (SleepyGemmy) [inline html/changelogs/wickedcybs_rename.yml:7]:
```suggestion
  - backend: "The Unathi tail sprite files don't reference their old name of 'Soghun' anymore."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm (hunk 2 above)
Comment (WickedCybs) [reply code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm:11]: Not really something I plan to do yeah.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**